Source/Turbo/ViewControllers/HotwireTabBarController.swift

```diff
@@ -25,17 +35,31 @@ open class HotwireTabBarController: UITabBarController {
     ///   used to handle `Navigator`'s requests and actions.
     ///
     /// This method assigns the provided tabs to the controller and sets up each tab's view controller.
-    public func load(_ tabs: [HotwireTab], navigatorDelegate: NavigatorDelegate? = nil) {
+    public func load(_ tabs: [HotwireTab]) {
         hotwireTabs = tabs
         viewControllers = tabs.map {
             setupViewControllerForTab($0, navigatorDelegate: navigatorDelegate)
         }
+        navigatorsByTab.forEach { tab, navigator in
+            navigator.route(tab.url)
+        }
```

Comment:
This ensures that the `viewControllers` property is set before routing any URLs. Before `viewControllers` is set `selectedIndex` is always `NSNotFound` which will cause a `fatalError` when accessing `activeNavigator`.